lume.lua

```diff
@@ -70,6 +70,16 @@ function lume.clamp(x, min, max)
 end
 
 
+function lume.floor(x)
+    return math_floor(x)
```

Comment:
@JosephShering, the rest of the file is also using 2 whitespaces for indentation. :)

@rxi, you may want to consider adding a [.editorconfig](http://editorconfig.org/) to point contributors to or add a [guideline](https://help.github.com/articles/setting-guidelines-for-repository-contributors/) to make mention of things like this.